docs/ENV.md

```diff
@@ -34,6 +44,27 @@ The database user.
 #### `CUBEJS_DB_PASS`
 The password for the database user.
 
+### Database settings (for BigQuery)
+You may wish to use BigQuery for your staging or production setup. However, if you want to connect from your local setup, you can do that as well. Configure the following env variables to connect your Cube.js instance to your BigQuery instance. These variables are not defined in `.env.example`.
```

Comment:
these variables should be defined in `.env.example` no?

Reply:
Added.